pages/rmrk/u/_id.vue

```diff
@@ -73,7 +73,7 @@
           <GalleryCardList
             :items="collections"
             type="collectionDetail"
-            link="rmrk/collection"
+            link="rmrk/collections"
```

Comment:
u sure?

Reply:
Good catch. This is my next plan to take a look. 
This is actually not just a problem with this collection.
I see that we need to improve LinkResolver and its usage in nuxt project.